weather_mv/loader_pipeline/sinks.py

```diff
@@ -163,8 +163,11 @@ def _get_band_data(i):
     ds = xr.merge(band_data_list)
     ds.attrs['is_normalized'] = ds_is_normalized_attr
 
-    end_time = None
-    if initialization_time_regex and forecast_time_regex:
+    start_time, end_time = None, None
+    time_attrs_present = 'start_time' in ds.attrs and 'end_time' in ds.attrs
+    if time_attrs_present:
+        start_time, end_time = ds.attrs['start_time'], ds.attrs['end_time']
+    elif initialization_time_regex and forecast_time_regex:
```

Comment:
Considering the user-specified flags `initialization_time_regex` and `forecast_time_regex`, we should prioritize the case where these flags are used over the case where `time_attrs_present` is used, and therefore move this case above the `time_attrs_present` case.

WDYT ?

Reply:
Can you explain by what you mean by prioritize? Shall I switch the order of the conditional blocks (user input gets checked first)?

Reply:
> Shall I switch the order of the conditional blocks (user input gets checked first)?

Yes, exactly. I think user inputs should be given priority over the default.